client/webserver/site/src/js/markets.ts

```diff
@@ -2027,51 +2077,175 @@ export default class MarketsPage extends BasePage {
     this.balanceWgt.updateAsset(this.openAsset.id)
   }
 
-  /* lotChanged is attached to the keyup and change events of the lots input. */
-  lotChanged () {
+  lotFieldKeyUpEvent () {
     const page = this.page
-    const lots = parseInt(page.lotField.value || '0')
-    if (lots <= 0) {
-      page.lotField.value = '0'
+
+    // We can't do anything with empty value here, and since it is a default
+    // we don't want to report is to be an issue either. Just make sure to
+    // reset both lots and quantity because both represent the same thing.
+    if (page.lotField.value === '') {
       page.qtyField.value = ''
-      this.previewQuoteAmt(false)
       return
     }
-    const lotSize = this.market.cfg.lotsize
-    page.lotField.value = String(lots)
-    // Conversion factor must be a multiple of 10.
-    page.qtyField.value = String(lots * lotSize / this.market.baseUnitInfo.conventional.conversionFactor)
-    this.previewQuoteAmt(true)
+
+    const [inputInvalid, adjusted,, adjQty] = this.parseLotInput()
+    if (inputInvalid || adjusted) {
+      // Let the user know that lot value he's entered was rounded down to the
+      // nearest integer number.
+      highlightOutlineRed(page.lotField)
+      highlightBackgroundRed(page.lostSizeBox)
```

Comment:
Sorry. I meant to do **just** the outline, at least for input elements. We don't need both.

Reply:
I highlight 2 different elements here (not same element twice):
- for `input` outline is highlighted
- for others it's background,

I think it makes sense to highlight `page.lostSizeBox` in addition to the field where user types his input, so that he gets a better sense of what's wrong,

maybe for **lot-input** of particular it's not very helpful, but when user enters **quantity=15** (and **lot size = 20**) highlighting lot size makes sense to me